### PR TITLE
Lombok

### PR DESCRIPTION
**Pull request by nareshk-gs**

### CODE REVIEW

File: src/main/java/org/sundry/recipes/bootstrap/RecipeBootstrap.java [hunk 1]
```diff
@@ -138,6 +141,7 @@ public void onApplicationEvent(ContextRefreshedEvent event) {
     guacRecipe.getCategories().add(mexicanCategory);
   
     recipes.add(guacRecipe);
+    log.debug("Recipe Guacamole is created");
```
Comment (pegwymonie) [inline src/main/java/org/sundry/recipes/bootstrap/RecipeBootstrap.java]: log.debug("Recipe {} Saved", guacRecipe.getName());

possibly refactor

```
public void saveRecipe(Recipe recipe){
  recipies.add(recipe);
  log.debug("Recipe {} saved", recipe.getName());
}
```

File: src/main/java/org/sundry/recipes/controllers/IndexController.java [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
+    log.debug("On the controller's constructor");
     this.recipeService = recipeService;
   }
   
   @RequestMapping({"","/"})
   public String getIndex(Model model) {
     model.addAttribute("recipes", recipeService.getRecipes());
-    
+    log.debug("Returning recipes to web UI");
     return "index";
```
Comment (pegwymonie) [inline src/main/java/org/sundry/recipes/controllers/IndexController.java]:
```
private static final String INDEX = "index";
...
return INDEX;
```

File: src/main/java/org/sundry/recipes/controllers/IndexController.java [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
   public IndexController(RecipeService recipeService) {
+    log.debug("On the controller's constructor");
     this.recipeService = recipeService;
   }
   
   @RequestMapping({"","/"})
   public String getIndex(Model model) {
     model.addAttribute("recipes", recipeService.getRecipes());
-    
+    log.debug("Returning recipes to web UI");
```
Comment (pegwymonie) [inline src/main/java/org/sundry/recipes/controllers/IndexController.java]:
```
List<Recipes> recipes = recipeService.getRecipes();
model.addAttribute("recipes", recipes);

"Returning {} recipes", recipes.count()
```

File: src/main/java/org/sundry/recipes/model/Category.java [hunk 4]
```diff
@@ -8,6 +10,8 @@
 /**
  * Created by kon1299 on 2019-06-25
  */
+@Data
+@EqualsAndHashCode(exclude = {"recipes"})
```
Comment (pegwymonie) [inline src/main/java/org/sundry/recipes/model/Category.java]: +1

File: src/main/java/org/sundry/recipes/model/Category.java [hunk 5]
```diff
@@ -8,6 +10,8 @@
 /**
  * Created by kon1299 on 2019-06-25
  */
+@Data
```
Comment (pegwymonie) [inline src/main/java/org/sundry/recipes/model/Category.java]: I would be surprised if this works with Hibernate. Most JPA frameworks like beans, IE public no args, setter based injection. You might need to add some annotations to get it to use this correctly.

Comment (nareshk-gs) [reply src/main/java/org/sundry/recipes/model/Category.java]: It works and application runs successfully in my local. Category has no other method except get and set. There is no constructor as well.

**Pull request merged**